src/refactor_nrepl/find/find_symbol.clj

```diff
@@ -103,41 +102,42 @@
          (str/join "\n")
          str/trim)))
 
-(defn- find-symbol-in-file [fully-qualified-name ignore-errors ^File file]
+(defn- find-symbol-in-file [fully-qualified-name ignore-errors referred-syms ^File file]
```

Comment:
as I understand, `referred-syms` doesn't have to be called each time. it can be calculated once in `find-global-symbol` and passed here. otherwise it's being unnecessarily called each time for each .clj file.